Argo/Functions/catDecoded.swift

```diff
@@ -1,3 +1,17 @@
+/**
+  Remove `Failure` values from an array and unwrap `Success`es.
+
+  This will iterate through the array of `Decoded<T>` elements and safely
+  unwrap the values.
+
+  If the element is `.Success(T)`, it will unwrap the value and add it into the
+  array.
+
+  If the element is `.Failure`, it will remove it from the array.
```

Comment:
remove it or just skip it ... the success route says it will "add it into the array." so should the failure part read more like "it will not add it to the array."?